ee/codegen/src/__test__/project.test.ts

```diff
@@ -253,17 +253,15 @@ describe("WorkflowProjectGenerator", () => {
       expectProjectFileToExist(project, ["workflow.py"]);
       expectProjectFileToMatchSnapshot(project, ["nodes", "bad_node.py"]);
 
-      expectErrorLog(
-        project,
-        `\
-Encountered 1 error(s) while generating code:
-
-- Failed to generate attribute 'BadNode.inputs.other': Failed to find node with id 'node_that_doesnt_exist'
-`
-      );
+      expect(project.workflowContext.getErrors()).toEqual([
+        new NodeAttributeGenerationError(
+          "Failed to generate attribute 'BadNode.inputs.other': Failed to find node with id 'node_that_doesnt_exist'",
+          "WARNING"
+        ),
+      ]);
     });
 
-    it("should fail to generate code if a node fails in strict mode", async () => {
```

Comment:
These were the file changes that had me starting to question whether we should actually keep these as errors and instead codegen in non-strict mode